src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -128,6 +128,16 @@ export default {
         });
     },
 
+    pushModifiedExpenseNotification({reportAction, onClick}, usesIcon = false) {
+        push({
+            title: _.map(reportAction.person, (f) => f.text).join(),
+            body: ReportUtils.getModifiedExpenseMessage(reportAction),
+            delay: 0,
+            onClick,
+            icon: usesIcon ? EXPENSIFY_ICON_URL : '',
```

Comment:
what kinds of notifications don't use the icon?

Reply:
Web does [here](https://github.com/Expensify/App/blob/89645c7e310ef497bbdebcb1c47c7527878e08de/src/libs/Notification/LocalNotification/index.website.js#L4) for comment notifications but desktop does not.

Reply:
Yeahh... I'd change it but I'm trying not to make this PR bigger than it already is 😅. Maybe just wait for the TS migration on this lib to clean it all up?